tests/markdown.js

```diff
@@ -0,0 +1,150 @@
+/*jshint multistr: true */
```

Comment:
wait, why is this needed? typically this was only necessary in old-style tests with inline XML strings.